app/components/contact-page-components/ErrorMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface ErrorMessageProps {
+  message: string;
```

Comment:
Do you think you should consider an _optional_ status code? If you do, make sure to factor it in your code as well. But great job making a reusable component! Excellent stuff!

Reply:
in this page I'm using the React.FC and react is making me import 'react'

import React from "react";

interface ErrorMessageProps {
  message: string;
}

const ErrorMessage: React.FC<ErrorMessageProps> = ({ message }) => {
  return (
    <div
      className={`absolute bottom-[-2rem] left-2 text-red-600 transition duration-300 ease-in dark:text-red-400 ${
        message ? "translate-y-0 opacity-100" : "translate-y-2 opacity-0"
      }`}
    >
      <div>{message}</div>
    </div>
  );
};

export default ErrorMessage;

Reply:
My question pertained to your Error Message. If you plan on showing any error or success codes, you will want to add this to your props as well. If you're unsure, you could always make it optional. What you have is perfectly fine.